boosts.js

```diff
@@ -12716,10 +12732,10 @@ Molpy.Coallate = function(){
 			if (this.bought == 1) {
 				this.countdown = 1000;
 				this.Unlock();
-			} else if (this.bought == 10) {
+			} else if (this.bought == this.limit) {
 				this.countdown = 0;
 				this.power = 0;
```

Comment:
I don't understand why we want the power = 0 but it isn't that terrible a loss of just over 154 power.